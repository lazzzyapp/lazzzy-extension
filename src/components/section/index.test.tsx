import Section from './index';
import React from 'react';
import { render } from 'enzyme';
import 'enzyme-adapter-react-16';

describe('test Section', () => {
  it.skip('should render correct', () => {
    expect(
      render(
        <Section title="test">
          <div>test</div>
        </Section>
      )
    ).toMatchSnapshot();
  });
});
