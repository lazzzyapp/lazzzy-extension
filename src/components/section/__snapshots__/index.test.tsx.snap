// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Section should render correct 1`] = `
initialize {
  "0": Node {
    "attribs": Object {},
    "children": Array [
      Node {
        "attribs": Object {
          "class": "sectionTitle",
        },
        "children": Array [
          Node {
            "data": "test",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "h1",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Node {
          "attribs": Object {},
          "children": Array [
            Node {
              "data": "test",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Node {
        "attribs": Object {},
        "children": Array [
          Node {
            "data": "test",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Node {
          "attribs": Object {
            "class": "sectionTitle",
          },
          "children": Array [
            Node {
              "data": "test",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "h1",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": Node {
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
    },
    "prev": null,
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
